docs/topics/commands.rst

```diff
@@ -271,11 +271,17 @@ crawl
 
 Start crawling using a spider.
 
+Supported options:
+
+* ``--output-format`` or ``-o/-O``: select output format.
```

Comment:
I don’t think we should only cover 2 options, as that is slightly worse than before. Before the documentation was incomplete, now it is misleading as users may think from reading the documentation that only these options are available. What would you think about covering all options of `scrapy crawl`?

Also mind that you are mixing 3 different options here. ``--output-format/-t` is 1 option, `-o` is another option, and `-O` is another option, and they should be covered separately.